sdk/core/azure-core/inc/azure/core/http/policies/policy.hpp

```diff
@@ -427,59 +427,4 @@ namespace Azure { namespace Core { namespace Http { namespace Policies {
         Context const& ctx) const override;
   };
 
-  namespace _internal {
-    /**
-     * @brief #Azure::Core::Http::Policies::_internal::ValuePolicy options.
-     */
-    struct ValueOptions
-    {
-      CaseInsensitiveMap HeaderValues;
-      std::map<std::string, std::string> QueryValues;
-    };
-
-    /**
-     * @brief Value policy.
-     *
-     * @details Applies key-value pair values to each HTTP request (either HTTP headers or query
-     * parameters).
-     */
-    class ValuePolicy : public HttpPolicy {
```

Comment:
FYI, this is the type of change we wouldn't (easily) be able to make post-GA. Even if we no longer have any users within the repo, if we GA'd a package that used it at some point, we'd have to keep such internal types around.